inst/templates/derivationchain/custom.js

```diff
@@ -160,6 +160,7 @@ var insertEdge = function(edgeList, fromId, toId, nodeData, traversedEdgeMap, in
 				target: toId, 
 				faveColor: color, 
 				strength: 20 ,
+				panningEnabled: false,
```

Comment:
this didn't really work, it was a fluke when i was testing that it somehow did..